package.json

```diff
@@ -121,7 +122,9 @@
       "packages/*"
     ],
     "globals": {
-      "react": "React"
+      "react": "React",
+      "react/jsx-runtime": "ReactJSX",
+      "react/jsx-dev-runtime": "ReactJSXDev"
```

Comment:
These don't actually exist but we still want UMD builds for Emotion so these are just to make it so Preconstruct can make a build